docker-compose.solr.yaml

```diff
@@ -23,7 +23,7 @@ services:
     volumes:
       - .:/mnt/ddev_config
       - solr:/var/solr
-    command: bash -c "
+    command: bash -c "set -eu
```

Comment:
OK, this doesn't work. If a collection already exists, it is fine. But with that option, it terminates with an error.

Reply:
You could consider using `solr create || true`. But however you like it. I just always want to know if the script is failing.